.github/CODEOWNERS

```diff
@@ -461,6 +461,7 @@ src/plugins/newsfeed @elastic/kibana-core
 test/common/plugins/newsfeed @elastic/kibana-core
 x-pack/plugins/notifications @elastic/appex-sharedux
 x-pack/test/cases_api_integration/common/plugins/observability @elastic/response-ops
+x-pack/plugins/observability @elastic/observability-ui
```

Comment:
I dont think this is a change we would want to be making here, right? Perhaps was a merge error that went unnoticed? 🤔

Reply:
Yes, @kibanamachine automatically added this line in https://github.com/elastic/kibana/pull/151445/commits/cacd82bc36c94de45bb779055c1241aba73368ba. I thought it might be necessary so I left it as it is. I will try deleting it and see if it will be added again.